### ISSUE TITLE
issue with mel_filters not present on non tiny-en models

### ISSUE THREAD

**Issue by ldenoue** [xenova/transformers.js#113]
**Describe the bug**
*A clear and concise description of what the bug is.*

Whisper Xenova models `Xenova/whisper-tiny.en` `Xenova/whisper-base.en` and `Xenova/whisper-base` fail at transformers.js

Note that `Xenova/whisper-tiny.en` works with the exact same audio file and pipeline.

![image](https://github.com/xenova/transformers.js/assets/149561/87d7e3a8-e7c1-4f0f-8216-d22bf3badb22)

![image](https://github.com/xenova/transformers.js/assets/149561/13d72fad-e596-428e-b144-f4dd7837a934)

**How to reproduce**
*Steps or a minimal working example to reproduce the behavior*

just any audio even short (6 seconds)

**Expected behavior**
*A clear and concise description of what you expected to happen.*

would like to avoid seeing the error :) 

**Logs/screenshots**
*If applicable, add logs/screenshots to help explain your problem.*

**Environment**
- Transformers.js version: lastest (I do `import { pipeline } from 'https://cdn.jsdelivr.net/npm/@xenova/transformers'` in a web worker on Chrome)
- Browser (if applicable): Chrome
- Operating system (if applicable): macOS 12.6 (21G115)
- Other:

**Additional context**
*Add any other context about the problem here.*

**Comment by kungfooman** [comment]
Hi, thank you for the bug report!

We just fixed it here: https://github.com/xenova/transformers.js/pull/112

You can use the latest version e.g. via ES6 + import maps.

TLDR they don't ship `mel_filters` in the .json file any longer, because they are completely defined by the configuration file itself already.

I would appreciate any feedback on the fix or if you notice any issues with it as there are tiny differences in the floating point approximation. :pray:
